toolkit/tools/roast/roast.go

```diff
@@ -51,12 +53,15 @@ var (
 	workers = app.Flag("workers", "Number of concurrent goroutines to convert with.").Default(defaultWorkerCount).Int()
 
 	imageTag = app.Flag("image-tag", "Tag (text) appended to the image name. Empty by default.").String()
+
+	timestampFile = app.Flag("timestamp-file", "File that stores timestamp for this program.").Required().String()
```

Comment:
Nit-pick: if we're storing multiple timestamps in the file, then let's use plural "timestamps".
```suggestion
	timestampFile = app.Flag("timestamp-file", "File that stores timestamps for this program.").Required().String()
```